packages/pelagos/src/listInput/ListInput.js

```diff
@@ -212,7 +215,9 @@ ListInput.propTypes = {
 	helperText: PropTypes.string,
 	/** The error text. */
 	error: PropTypes.string,
-	/** Whether the list should be displayed as columns. */
+	/** Whether the list should be reorderable. */
+	reorderable: PropTypes.bool,
+	/** Whether the list should be displayed as columns. Defaults to true if reorderable */
```

Comment:
Same notes as before:
- Maybe "Set to true..." as I think "Defaults..." may imply that the developer can override it (which is not the case).
- The dot at the end is missing.